src/components/Group/Group.tsx

```diff
@@ -0,0 +1,40 @@
+import classNames from "classnames"
+import { Card } from "../Card"
+
+type ElementProps = {
```

Comment:
Niepotrzebnie, mamy już stworzony typ, w komponencie Card, te elementy to Twoje Card, więc ich typem powinien być CardProps

Reply:
Trochę takie noob question. `interface CardProps` przenieść do folderu `types` i pliku `types.ts`?, czy w `Card.tsx` dodać `export` do `interface CardProps`?.